core/src/main/java/org/apache/brooklyn/core/mgmt/entitlement/WebEntitlementContext.java

```diff
@@ -21,15 +21,24 @@
 import org.apache.brooklyn.api.mgmt.entitlement.EntitlementContext;
 import org.apache.brooklyn.util.javalang.JavaClassNames;
 
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * Indicates an authenticated web request as the entitlements context;
  * note user may still be null if no authentication was requested
  */
 public class WebEntitlementContext implements EntitlementContext {
 
+    public static final String ENTITLEMENTS_ATTRIBUTES = "brooklyn.entitlements.attributes";
+
+    public static final String USER_ROLES = "brooklyn.entitlements.user.roles";
```

Comment:
are these `ROLES` or `GROUPS` ?  when we fetch we call them groups.  should be consistent everywhere.  personally i prefer groups.  maybe worth a comment in the code to indicate these are sometimes in LDAP called roles (if that's true?).

Reply:
Groups is better. Until the RoleResolver is executed there is not direct relation between groups and roles. Changing it.